fdbserver/Resolver.actor.cpp

```diff
@@ -418,7 +422,7 @@ ACTOR Future<Void> resolveBatch(Reference<Resolver> self, ResolveTransactionBatc
 						writtenTLogs.insert(i);
 					}
 				} else {
-					toCommit.getLocations(reply.writtenTags, writtenTLogs);
+					toCommit->getLocations(reply.writtenTags, writtenTLogs);
```

Comment:
It appears this code block is not guarded by "PROXY_USE_RESOLVER_PRIVATE_MUTATIONS" check (only checks if "ENABLE_VERSION_VECTOR_TLOG_UNICAST" is enabled). Is that good enough?

Reply:
UNICAST can only be set if RESOLVE_PRIVATE_MUTATIONS is also set, although that is probably not obvious to someone unfamiliar to the code.

Reply:
> UNICAST can only be set if RESOLVE_PRIVATE_MUTATIONS is also set, 

Right, but do we have any explicit check in the code that ensures that?

Reply:
I don't think we have checks for that. Probably better to set the knob if unicast is enabled.